inc/fields/textarea-field.class.php

```diff
@@ -21,7 +21,7 @@ public function displayField($canEdit = true)
          if ($CFG_GLPI["use_rich_text"]) {
             echo plugin_formcreator_decode($this->getAnswer());
          } else {
-            echo nl2br($this->getAnswer());
+            echo str_replace(array("\\r\\n", "\\r", "\\n"), "<br />", $this->getAnswer());
```

Comment:
i'm not very pleased by this change.
If possible, we should store the data without double antislash.
nl2br is a good function, imo